.github/workflows/non-cot-pr-build-and-e2e-tests.yml

```diff
@@ -66,8 +65,8 @@ jobs:
                   if-no-files-found: ignore
                   retention-days: 5
 
-    cot-api-tests-run:
-        name: Runs API tests with COT enabled.
+    non-cot-api-tests-run:
```

Comment:
```
non-hpos-api-tests-run:
```

Reply:
Marking this conversation as resolved now, since I can already see the suggested change.